packages/issuer-sdk-js/README.md

```diff
@@ -150,16 +157,33 @@ const idos = await idOS.init(...);
 // some endpoint you expose. But, to keep it simple, we're using a constant.
 const ISSUER_SIGNER_ADDRESS = "0xc00ffeec00ffeec00ffeec00ffeec00ffeec00ff";
 
-// Ask the user for a Write Grant
-await idos.data.addWriteGrant(ISSUER_SIGNER_ADDRESS);
+const { signMessageAsync } = useSignMessage();
+const currentTimestamp = Date.now();
+const currentDate = new Date(currentTimestamp);
+const notUsableAfter = new Date(currentTimestamp + 24 * 60 * 60 * 1000);
+const delegatedWriteGrant = {
+  owner_wallet_identifier: address as string,
+  grantee_wallet_identifier: ISSUER_SIGNER_ADDRESS,
+  id: crypto.randomUUID(),
+  access_grant_timelock: currentDate.toISOString().replace(/.\d+Z$/g, "Z"),  // Need to cut milliseconds to have 2025-02-11T13:35:57Z datetime format
+  not_usable_before: currentDate.toISOString().replace(/.\d+Z$/g, "Z"),
+  not_usable_after: notUsableAfter.toISOString().replace(/.\d+Z$/g, "Z"),
+};
+
+// Get a message to sign
+const message: string = await clientSDK.data.requestDWGSignature(delegatedWriteGrant);
+
+// Ask a user to signt the message.
+const signature = await signMessageAsync({ message });
```

Comment:
```suggestion
const signature = await signer.signMessage(message);
```

This should be more obvious to anyone familiar with EVMs.

Reply:
Good point. Done.